src/app/examples/empty-state-example/empty-state-example.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'kirby-empty-state-example',
+  templateUrl: './empty-state-example.component.html',
+  styleUrls: ['./empty-state-example.component.scss'],
+})
+export class EmptyStateExampleComponent {
+  iconName = 'help';
```

Comment:
Consider inlining these variables, as it will make the example code more clear

Reply:
Fixed